source/guide/3.2/reference-builtin-workflows.md

```diff
@@ -192,3 +192,49 @@ This sub-graph determines the operations that will be executed during the workfl
 * The following node instances will be re-installed: `war_1`, `webserver_1` and `webserver_host_1`.
 * The following relationships will be re-established: `war_1` **connected to** `database_1` and `webserver_host_1` **connected to** `floating_ip_1`.
 {%endnote%}
+
+# Scale
+
+**Workflow name:** *scale*
+
+**Workflow description:**
+
+Scales out/in the node subgraph of the system topology applying the `install`/`uninstall` workflows' logic respectively.
+
+If the node denoted by `node_id` is contained in a compute node (or is a compute node itself) and `scale_compute` is `true` (which is the default),
+the node graph will consist of all nodes that are contained in the compute node which contains `node_id` and the compute node itself.
```

Comment:
this is a bit confusing, how about just saying that the root of the subgraph in this case will be the compute node, and else it will be the node itself. and move the note from below to here